conda-envs/environment-dev-py36.yml

```diff
@@ -1,31 +1,22 @@
 name: pymc3-dev-py36
 channels:
-  - conda-forge
-  - defaults
+- conda-forge
+- defaults
 dependencies:
-  - python=3.6
-  - arviz>=0.9
-  - theano-pymc==1.0.12
-  - numpy>=1.13
-  - scipy>=0.18
-  - pandas>=0.18
-  - patsy>=0.5
-  - fastprogress>=0.2
-  - h5py>=2.7
-  - typing-extensions>=3.7
-  - python-graphviz
-  - ipython>=7.16
-  - nbsphinx>=0.4
-  - numpydoc>=0.9
-  - pre-commit>=2.8.0
-  - pytest-cov>=2.5
-  - pytest>=3.0
-  - recommonmark>=0.4
-  - sphinx-autobuild>=0.7
-  - sphinx>=1.5
-  - watermark
-  - dataclasses  # python_version < 3.7
-  - contextvars  # python_version < 3.7
-  - mkl-service
-  - dill
-  - libblas=*=*mkl
+- contextvars
+- dataclasses
+- h5py>=2.7
+- ipython>=7.16
+- libblas=*=*mkl
```

Comment:
I think these two should move to the standard requirements too, no?

Reply:
as in, `ipython` and `libblas`?

I don't think `ipython` is a hard requirement (is it?), and I don't think you can install `libblas` with `pip` (I figure that's the reason conda is preferred for installing requirements here?)